### DISCUSSION TITLE
Infotable Technology

### DISCUSSION THREAD

**Discussion by Saturo78** [streetcomplete/StreetComplete#4364] [Ideas]
When recording infotables, there should be the option "technology".

**Comment by westnordost** [comment]
What is an infotable?

**Reply by Saturo78** [reply]
Maybe you might name it Infobord or something else.
In which category goes technical information like about a lock or dock?

**Reply by legofahrrad** [reply]
@westnordost 
I guess the feature request is about adding an option "technology" to the `board_type` quest.

**Comment by legofahrrad** [comment]
`technology` as an option was rejected in #3171 due to rare use. You can leave a note in case of a technology information board.

**Comment by Saturo78** [comment]
Really, technology is to rare but we have plants, animals and nature?

**Reply by Helium314** [reply]
Technology has 433 uses, plants has 6467, nature has 38082.
"Animals" can't be selected in SC (probably because it only has 6 uses).

**Comment by Saturo78** [comment]
Kein Problem, ich verwende einfach Vespucci, damit ist es problemlos möglich.

Wo liegt das Problem einfach eine weitere Eingabemöglichkeit zuzulassen?
